test/default-messages.js

```diff
@@ -1,37 +1,31 @@
 var tap = require('tap');
 var spawn = require('child_process').spawn;
-var trim = require('string.prototype.trim');
+var concat = require('concat-stream');
 
 tap.test('default messages', function (t) {
     t.plan(1);
 
-    var tc = tap.createConsumer();
+    var ps = spawn(process.execPath,
+        [ require('path').join(__dirname, 'messages', 'defaults.js') ]);
```

Comment:
Good point! 
Went with the single line with the `require('path') moved above its only 89 chars which I think is _acceptable_:
```js
var ps = spawn(process.execPath, [path.join(__dirname, 'messages', 'defaults.js')]);
```